inst/rmarkdown/templates/flex_dashboard/resources/flexdashboard.js

```diff
@@ -138,12 +139,12 @@ var FlexDashboard = (function () {
     initPrismHighlighting();
 
     // record mobile and orientation state then register a handler
-    // to refresh if it changes
+    // to refresh if auto_reload is set to true and it changes
     _options.isMobile = isMobilePhone();
     _options.isPortrait = isPortrait();
     $(window).on('resize', function() {
-      if (_options.isMobile !== isMobilePhone() ||
-          _options.isPortrait !== isPortrait()) {
+      if ((_options.isMobile !== isMobilePhone() && _options.auto_reload) ||
```

Comment:
You could enclose the entire body of the function in `_options.auto_reload` (or even skip adding the resize handler altogether if it's `false`)

Reply:
Ok, will do.

Reply:
Since the name of the argument changed, I changed the name of the pull request for consistency. 
Changes are now in #247